tests/parse-includes/compiler_output.txt

```diff
@@ -1,5 +1,5 @@
 version.cpp
-Note: including file: C:\Users\me\test\smartsqlite\include\smartsqlite/version.h
+Note: including file: C:\Projects\test\smartsqlite\include\smartsqlite/version.h
```

Comment:
This part I don't get -- why did you replace `Users\me` with `Projects` all over?